### ISSUE TITLE
Make serialization consistant

### ISSUE THREAD

**Issue by fredkingham** [openhealthcare/opal#1494]
a patient to_dicted contains empty lists for patient subrecords that are not there.

An episode to_dicted does not.

an Episode.objects.serialised, does.

consistency please.

**Comment by fredkingham** [comment]
e.g. for a patient with no allergies.
```
self.assertTrue("allergy" in patient.to_dict(None))
self.assertFalse("allergy" in episode.to_dict(None))
self.assertTrue("allergy" in Episode.objects.serialised([episode], None))
```

**Comment by davidmiller** [comment]
I think this is a bug and that the empty key should always be there.
Can remember when this caused issues with javascript Episode objects.

**Comment by davidmiller** [comment]
Given that we've merged #1503 I think this is only still open b/c 0.11.0 isn't yet the default branch ?

Closing this now...
